### DISCUSSION TITLE
Help: Unable to build tasmota-sensors with TLS

### DISCUSSION THREAD

**Discussion by g0lgs** [arendst/Tasmota#12937] [Q&A] [answered]
If I try to build tamsota-sensors variant with TLS enabled I get: 

tasmota/WiFiClientSecureLightBearSSL.h:30:23: fatal error: t_bearssl.h: No such file or directory

yet I can build the 'tamsota' variant with the same user_config_override.h without any issues.

**Comment by sfromis** [comment, marked as answer]
Yes, that's by design. As [documented](https://tasmota.github.io/docs/Compile-your-build/#customize-your-build), the standard build setup does not allow you to add features to the preselected mixes like tasmota-sensors, only to the base "tasmota" build environment.

**Reply by g0lgs** [reply]
Ok I missed that point.

So I need to fathom out from my_user_config.h which combination of options I need for the sensor(s) I want to support then ?

**Reply by sfromis** [reply]
Maybe easier to check [BUILDS](https://github.com/arendst/Tasmota/blob/development/BUILDS.md).
